src/devhub/entity/proposal/Proposal.jsx

```diff
@@ -0,0 +1,1262 @@
+    },
+  },
+  {
+    label: "Canceled",
+    value: {
+      status: TIMELINE_STATUS.CANCELED,
+      sponsor_requested_review: false,
+      reviewer_completed_attestation: false,
+    },
+  },
+  {
+    label: "Payment-processing",
+    value: {
+      status: TIMELINE_STATUS.PAYMENT_PROCESSING,
+      kyc_verified: false,
+      test_transaction_sent: false,
+      request_for_trustees_created: false,
+      sponsor_requested_review: true,
+      reviewer_completed_attestation: false,
+    },
+  },
+  {
+    label: "Funded",
+    value: {
+      status: TIMELINE_STATUS.FUNDED,
+      trustees_released_payment: true,
+      kyc_verified: true,
+      test_transaction_sent: true,
+      request_for_trustees_created: true,
+      sponsor_requested_review: true,
+      reviewer_completed_attestation: false,
+    },
+  },
+];
+
+const LinkedProposals = () => {
+  const linkedProposalsData = [];
+  snapshot.linked_proposals.map((item) => {
+    const data = Near.view("${REPL_DEVHUB_CONTRACT}", "get_proposal", {
+      proposal_id: item,
+    });
+    if (data !== null) {
+      linkedProposalsData.push(data);
+    }
+  });
+
+  return (
+    <div className="d-flex flex-column gap-3">
+      {linkedProposalsData.map((item) => (
+        <div className="d-flex gap-2">
+          <Widget
+            src={"${REPL_DEVHUB}/widget/devhub.entity.proposal.Profile"}
+            props={{
+              accountId: item.snapshot.editor_id,
+            }}
+          />
+          <div className="d-flex flex-column" style={{ maxWidth: 250 }}>
+            <b className="text-truncate">{item.snapshot.name}</b>
+            <div className="text-sm text-muted">
+              created on {readableDate(item.snapshot.timestamp / 1000000)}
+            </div>
+          </div>
+        </div>
+      ))}
+    </div>
+  );
+};
+
+const CheckBox = ({ value, isChecked, label, disabled, onClick }) => {
+  return (
+    <div className="d-flex gap-2 align-items-center">
+      <input
+        class="form-check-input"
+        type="checkbox"
+        value={value}
+        checked={isChecked}
+        disabled={disabled}
+        onChange={(e) => onClick(e.target.checked)}
+      />
+      <label style={{ width: "90%" }} class="form-check-label text-black">
+        {label}
+      </label>
+    </div>
+  );
+};
+
+const RadioButton = ({ value, isChecked, label }) => {
+  return (
+    <div className="d-flex gap-2 align-items-center">
+      <input
+        class="form-check-input"
+        type="radio"
+        value={value}
+        checked={isChecked}
+        disabled={true}
+      />
+      <label class="form-check-label text-black">{label}</label>
+    </div>
+  );
+};
+
+const tokenMapping = {
+  NEAR: "NEAR",
+  USDT: {
+    NEP141: {
+      address: "usdt.tether-token.near",
+    },
+  },
+  USDC: {
+    NEP141: {
+      address:
+        "17208628f84f5d6ad33f0da3bbbeb27ffcb398eac501a31bd6ad2011e36133a1",
+    },
+  },
+  OTHER: "OTHER",
+};
```

Comment:
The addresses are not needed anymore, just pass one of `"NEAR"`, `"USDT"`, `"USDC"`, or `"OTHER"` to `requested_sponsorship_paid_in_currency`

Reply:
oh I see, I will update